clojure-lang/.gitignore

```diff
@@ -0,0 +1,11 @@
+/target
+/classes
+/checkouts
+pom.xml
+pom.xml.asc
+*.jar
+*.class
+/.lein-*
+/.nrepl-port
+.hgignore
+.hg/
```

Comment:
This is not a mercurial project.

Reply:
@josiahcarlson  Sorry I didn't understand this comment. Can you please explain this bit more?